app/domain/operations/fdsh/payload_eligibility/check_base_eligibility_rules.rb

```diff
@@ -14,6 +14,7 @@ class CheckBaseEligibilityRules
           ssa: [:validate_ssn],
           dhs: [:validate_vlp_documents],
           local_residency: [],
+          alive_status: [:validate_ssn],
```

Comment:
All rules for alive_status verification should be listed here
```suggestion
          alive_status: [:validate_ssn, :is_member_enrolled?], # method name something like this
          income: [:validate_ssn],
```

Reply:
Going to revisit on when handling dmf response